.eslintrc.json

```diff
@@ -163,7 +163,8 @@
         "no-shadow": 2,
         "no-undef-init": 2,
         "no-undef": 2,
-        "no-unused-vars": 2,
+        "no-unused-vars": "off",
```

Comment:
It would be better if you just remove this rule? because we have another rule with the same functionality